src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -36,60 +47,322 @@ $pf-c-tree-view--MaxNesting: 10;
   --pf-c-tree-view__node-toggle-button--MarginTop: calc(var(--pf-global--spacer--form-element) * -1);
   --pf-c-tree-view__node-toggle-button--MarginBottom: calc(var(--pf-global--spacer--form-element) * -1);
 
-  // check
+  // Check
   --pf-c-tree-view__node-check--MarginRight: var(--pf-global--spacer--sm);
 
-  // badge
+  // Badge
   --pf-c-tree-view__node-count--MarginLeft: var(--pf-global--spacer--sm);
   --pf-c-tree-view__node-count--c-badge--m-read--BackgroundColor: var(--pf-global--disabled-color--200);
 
-  // search
+  // Search
   --pf-c-tree-view__search--PaddingTop: var(--pf-global--spacer--sm);
   --pf-c-tree-view__search--PaddingRight: var(--pf-global--spacer--sm);
   --pf-c-tree-view__search--PaddingBottom: var(--pf-global--spacer--sm);
   --pf-c-tree-view__search--PaddingLeft: var(--pf-global--spacer--sm);
 
-  // icon
+  // Icon
   --pf-c-tree-view__node-icon--PaddingRight: var(--pf-global--spacer--sm);
   --pf-c-tree-view__node-icon--Color: var(--pf-global--icon--Color--light);
+  --pf-c-tree-view__node-toggle-icon--base--Rotate: 0;
+  --pf-c-tree-view__node-toggle-icon--Rotate: var(--pf-c-tree-view__node-toggle-icon--base--Rotate);
   --pf-c-tree-view__list-item--m-expanded__node-toggle-icon--Rotate: 90deg;
 
-  // text
+  // Text
   --pf-c-tree-view__node-text--max-lines: 1;
 
-  // action
+  // Title
+  --pf-c-tree-view__node-title--FontWeight: var(--pf-global--FontWeight--bold);
+
+  // Action
   --pf-c-tree-view__action--MarginLeft: var(--pf-global--spacer--md);
   --pf-c-tree-view__action--focus--BackgroundColor: var(--pf-global--BackgroundColor--200);
   --pf-c-tree-view__action--Color: var(--pf-global--icon--Color--light);
   --pf-c-tree-view__action--hover--Color: var(--pf-global--icon--Color--dark);
   --pf-c-tree-view__action--focus--Color: var(--pf-global--icon--Color--dark);
 
+  // Guides
+  // ================================================================== //
+  --pf-c-tree-view--m-guides--border--Left: var(--pf-c-tree-view--m-guides--base-border--Left); // starting value, this gets updated for each nesting level
+  --pf-c-tree-view--m-guides--border--ZIndex: var(--pf-c-tree-view--m-guides--base-border--ZIndex);
+
+  // Base
+  --pf-c-tree-view--m-guides--base-border--BorderColor: var(--pf-global--BorderColor--100);
```

Comment:
just curious why these aren't named `--pf-c-tree-view--m-guides--border-color--base`, `--pf-c-tree-view--m-guides--border-width--base`, etc.

Reply:
also if we keep `border` in the var name, should that be guide?

Reply:
udpated